src/modules/languages/python.nix

```diff
@@ -21,5 +23,10 @@ in
     ];
 
     env.PYTHONPATH = "${config.env.DEVENV_PROFILE}/${cfg.package.sitePackages}";
+
+    enterShell = lib.mkIf cfg.venv.enable ''
+      python -m venv ${config.env.DEVENV_STATE}/venv
```

Comment:
This takes about 4-5s on my machine, should we skip it if it exists?

Reply:
I tried the following:

```
time python -m venv ${config.env.DEVENV_STATE}/venv
```

```
real	0m1.293s
user	0m1.230s
sys	0m0.063s
```

It wasn't that noticeable on my laptop, but it is indeed unnecessary. I've added a check so that it only creates venv if the directory doesn't exist. :+1: